providers/base/units/stress/s3s4.pxu

```diff
@@ -150,3 +150,221 @@ requires: cpuinfo.platform in ("i386", "x86_64")
 command: [ -e "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log ] && cat "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log
 _description:
  Attaches the log from the {s4_iterations} cycles Hibernate/Resume test if it exists
+
+id: stress_s3_cycles_iterations_1
+category_id: stress-tests/suspend
+summary:
+  Resource job meant to be used in the S3 cycles stress templates
+plugin: resource
+environ: STRESS_S3_ITERATIONS STRESS_SUSPEND_REBOOT_ITERATIONS
```

Comment:
Is the `STRESS_S3_ITERATIONS` used anywhere?

Reply:
Yes, some part is used to iterate and other is used to be a variable only(test case id)

Reply:
Could you point me to exact line where it is used? 
In this job the only envvar that is in use is the `STRESS_SUSPEND_REBOOT_ITERATIONS`.
If you mean that it is used by other jobs, then it's not necessary to declare them in other jobs.